ClientHints.php

```diff
@@ -188,10 +188,12 @@ public function getOperatingSystemVersion(): string
     public function getBrandList(): array
     {
         if (\count($this->fullVersionList)) {
-            return \array_combine(
-                \array_column($this->fullVersionList, 'brand'),
-                \array_column($this->fullVersionList, 'version')
-            ) ?: [];
+            $brands = \array_column($this->fullVersionList, 'brand');
```

Comment:
= $this->fullVersionList['brand'] ?? []

Reply:
That isn't the same. It extracts the `brand` column form all arrays contained in `$this->fullVersionList`